packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -61,6 +61,14 @@ Favoriting or other optional actions can be associated with any menu item. In th
 
 Any action that can be represented as an icon button can be placed in one or more menu items. In this case, the favoriting action is used to mark an item as a favorite.
 
+### Drilldown menu
+<img src="./img/drill-down-menu.png" alt="drilldown menu" width="935"/>
+
+Use a drilldown menu when data is structured into levels and includes a long list of options. When the parent with children is selected, the list is replaced with children items. A header displays the name of a parent with the option going one level back. 
+
+If data is more complex and has more than 2 levels, use drilldown menu with breadcrumbs instead. Breadcrumbs offer better navigation between different levels. The number of visited levels is shown in a grey badge with a dropdown menu showing the menu items between the first and last level. Badge allows you to go back to a specific visited level. If you need to see the whole structure of your data, use [tree view](/components/tree-view) instead. 
```

Comment:
Small edit #1: use **a** drilldown menu with breakdcrumps..." 

Small edit #2: Add "The" in front of "badge allows you to go back to a specific visited level"

Small edit #3: "use **a** tree view instead"